docs/introduction/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Introduction to Blazor
+author: guardrex
+description: Learn how Blazor runs in the browser to execute C#/Razor code with WebAssembly and the Mono runtime in this introduction.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/09/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/introduction/index
+---
+# Introduction to Blazor
+
+By [Steve Sanderson](http://blog.stevensanderson.com), [Daniel Roth](https://github.com/danroth27), and [Luke Latham](https://github.com/guardrex)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+Blazor is an experimental .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly. Blazor provides all of the benefits of a client-side web UI framework using .NET on the client and optionally on the server.
+
+## Why use .NET for client-side apps?
```

Comment:
> Why use .NET for client-side apps?

I'd like to clarify that this about client side *web* apps running in the browser. How about: "Why use .NET in the browser?" or "Why use .NET for client-side web apps?"